src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -28,6 +30,26 @@ public class GenerateConfig extends BaseCommand {
       description = "Directory to put generated cromwell.conf in. Defaults to current directory.")
   public String dir;
 
+  // We create an exclusive ArgGroup because we require one of either
+  // workspace_bucket_name or google_bucket_name.
+  @CommandLine.ArgGroup(exclusive = true, multiplicity = "1")
+  WorkspaceOrGoogleBucketName workspace_or_google_bucket_name;
+
+  static class WorkspaceOrGoogleBucketName {
+    @CommandLine.Option(
+        names = "--workspace-bucket-name",
+        required = true,
+        description = "Terra workspace bucket to hold Cromwell log files.")
```

Comment:
It's more than just log files, right? It's Cromwell's workflow execution directory that stores log files, intermediate outputs, etc.

Unfortunately, the Cromwell documentation for the `root` field is not great:

https://cromwell.readthedocs.io/en/stable/backends/Google/

The best I can suggest here is that we say:

> Terra workspace bucket used as the root of Cromwell workflow execution.

Please update language throughout.

Reply:
Yea, I like your suggestion better. Fixed.